packages/lib/gpcircuits/test/multituple.spec.ts

```diff
@@ -0,0 +1,258 @@
+import { expect } from "chai";
+import { WitnessTester } from "circomkit";
+import "mocha";
+import { poseidon2, poseidon3, poseidon4 } from "poseidon-lite";
+import {
+  computeTupleIndices,
+  hashTuple,
+  maxTupleArity,
+  multiTupleHasher,
+  padArray,
+  requiredNumTuples,
+  tupleHasher,
+  MultiTupleModuleInputNamesType,
+  MultiTupleModuleInputs,
+  MultiTupleModuleOutputNamesType,
+  MultiTupleModuleOutputs
+} from "../src";
+import { circomkit } from "./common";
+import { PODValue, podValueHash } from "@pcd/pod";
+import { BABY_JUB_NEGATIVE_ONE } from "@pcd/util";
+
+describe("MultiTuple helpers should work", function () {
```

Comment:
Kudos for thorough tests of the helpers. I came to check as I read the code for them.  It's easier to evaluate coverage of edge cases in tests than by code inspection.